### ISSUE TITLE
Accomodate for additional elements

### ISSUE THREAD

**Issue by brasmus** [metno/esd#117]
We have tailored 'esd' for standard elements such as temperature and precip

These are available at http://eca.knmi.nl/dailydata/predefinedseries.php
e.g.
Daily maximum temperature TX (339Mb)
Daily minimum temperature TN (339Mb)
Daily mean temperature TG (320Mb)
Daily precipitation amount RR (503Mb)

There are also additional elements:	
Daily mean sea level pressure PP (91Mb)
Daily cloud cover CC (118Mb)
Daily humidity HU (115Mb)
Daily snow depth SD (217Mb)
Daily sunshine duration SS (73Mb)
Daily mean wind speed FG (54Mb)
Daily maximum wind gust FX (43Mb)
Daily wind direction DD (31Mb)

Should we also expand 'esd' for these (perhaps it's already done?)?

**Comment by abdelkaderm** [comment]
Yes, we need to update the glossary file accordingly, perhaps, write few functions that will convert standard elements for each data source into the common meta data ... i.e. search for data availability if the element has not been one of the default ones !

**Comment by abdelkaderm** [comment]
or write conversion function to deal with meta data from the web as well ! in the same way station works, instead of creating `metadata` object
